trunk build failing due to mccs checksum mismatch

Getting this in opam-ci when rebuilding opam

```
rm -rf jbuilder
MD5 for mccs.tar.gz differ:
  expected: dbbe94d760ddb89217d617041d12e789
    actual: 987c4970715713f21e6c79b4f3cba430
MD5 for mccs.tar.gz differ:
  expected: dbbe94d760ddb89217d617041d12e789
    actual: f48a70649b86c40ebb4c86ff0ce8a070
rm jbuilder.download
make[2]: Leaving directory '/tmp/opam/src_ext'
make[2]: *** [Makefile:152: mccs.download] Error 1
make[1]: *** [Makefile:54: lib-ext] Error 2
make[1]: Leaving directory '/tmp/opam'
make: *** [Makefile:191: cold] Error 2
```

--- a/src_ext/Makefile.sources
+++ b/src_ext/Makefile.sources
@@ -34,7 +34,7 @@ MD5_dose3 = e7d4b1840383c6732f29a47c08ba5650
 $(call PKG_SAME,dose3)
 
 URL_mccs = https://github.com/AltGr/ocaml-mccs/archive/1.1+8.tar.gz
-MD5_mccs = dbbe94d760ddb89217d617041d12e789
+MD5_mccs = 987c4970715713f21e6c79b4f3cba430
 
 $(call PKG_SAME,mccs)
 

